app/src/main/java/com/fightpandemics/home/HomeActivity.kt

```diff
@@ -17,10 +20,21 @@ class HomeActivity : BaseActivity(), HomeContract.View {
     @Inject
     lateinit var presenter: HomePresenter
 
+    private var isFabOpen = false
+
+    private lateinit var fab_open : Animation
+    private lateinit var fab_close : Animation
+    private lateinit var rotate_forward : Animation
+    private lateinit var rotate_backward : Animation
```

Comment:
To follow the name convention, I'd suggest using the _camelCase_ instead of the _snake_case_.